### PR TITLE
updated examples with react v18

### PR DESCRIPTION
**Pull request by joshua-barnard-evolv**
I've added my new examples that use React 18. They work on their own outside of this repo. We need to get them working within the mono-repo. NextJS works (I think), but the React example gets a conflict.

It'll need to be fixed before it's merged.

### CODE REVIEW

File: examples/nextjs/package.json [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
+  },
+  "devDependencies": {
+    "@types/node": "18.11.2",
+    "@types/react": "18.0.21",
+    "@types/react-dom": "18.0.6",
+    "eslint": "8.25.0",
+    "eslint-config-next": "12.3.1",
+    "typescript": "4.8.4"
```
Comment (mattstrom) [inline examples/nextjs/package.json:24]: These *technically* shouldn't be necessary. This repo uses workspaces, so each subproject will inherit dependencies from the root project.

File: examples/react/package.json [hunk 2]
```diff
@@ -1,30 +1,44 @@
...
-  "type": "module",
+  "dependencies": {
+    "@evolv/react": "^1.0.7",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.11.68",
+    "@types/react": "^18.0.21",
+    "@types/react-dom": "^18.0.6",
```
Comment (mattstrom) [inline examples/react/package.json:13]: Not critically important to change, but a lot of these are dev dependencies.

File: examples/react/src/index.tsx [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
 
 
-const rootElement = document.getElementById('root');
+const root = ReactDOM.createRoot(
+  document.getElementById('root') as HTMLElement
+);
 
-const options: EvolvClientOptions = JSON.parse(process.env.REACT_APP_EVOLV_CONFIG ?? '{}');
+const options: EvolvClientOptions = {
+  environment: "a925240014"
```
Comment (mattstrom) [inline examples/react/src/index.tsx:12]: Is this the environment ID for a common example project?

Comment (joshua-barnard-evolv) [reply examples/react/src/index.tsx:12]: I created a dedicated project in its own environment to use with these examples.

**Comment by joshua-barnard-evolv** [comment]
I made all the changes you requested. It's still really broken. We might need to troubleshoot this together.

**Comment by mattstrom** [comment]
I upgraded to React 18 in #29 and updated examples to reflect that. So I'm closing this as it is likely irrelevant now.

**Pull request closed without merging**